modules/menu/menu.c

```diff
@@ -1093,10 +1097,13 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 		/* stop any ringtones */
 		menu.play = mem_deref(menu.play);
 
-		/* Only play the ringtones if answermode is "Manual".
+		/* Only play the ringtones
+		 * if answermode is "Manual" or "Intercom".
 		 * If the answermode is "auto" then be silent.
 		 */
-		if (ANSWERMODE_MANUAL == account_answermode(ua_account(ua))) {
+		amode = account_answermode(ua_account(ua));
+		if ((ANSWERMODE_MANUAL == amode) ||
+			(ANSWERMODE_INTERCOM == amode)) {
```

Comment:
are you sure that we should play the ringtone if answermode=intercom ?

the call will be answered immediately, so the user will not hear any ringtone.

Reply:
I'm sure:
1. If we implement async timer, for delay before accept call.
2. `answer-after` param is not mandatory, if a regular call comes we need a way to alert user.

Reply:
Maybe it make sense, if we will play another sound, especially for intercom mode (if answer-after>0 is arrived) and default callwaiting.wav for a regular call.